siliconcompiler/report/streamlit_viewer.py

```diff
@@ -8,6 +8,7 @@
 import argparse
 import json
 import pandas
+import altair
```

Comment:
new dependency?

Reply:
Yes. Do I need to add this to the requirements.txt?

Reply:
yes